packages/server/billing/stripeWebhookHandler.ts

```diff
@@ -30,6 +30,20 @@ const eventLookup = {
         }
       `
     },
+    payment_succeeded: {
```

Comment:
`upgradeToTeamTier` is now called by Stripe's webhook rather having two mutations on the client. This should handle the use case where the client creates the stripe subscription but loses their internet connection before the upgrade happens

Reply:
@Dschoordsch would it be OK if I create an issue to handle this, ship the checkout flow, and then work on it right away?